openqa_review/openqa_review.py

```diff
@@ -1423,9 +1457,16 @@ def _next_label(self):
     def __str__(self):
         """Generate markdown."""
         report_str = ""
+        if self.args.todo_only:
+            report_str = "**TODO: review**\n\n"
         for k, v in self.report.items():
             if not self.args.skip_passed or type(v) is not ProductReport or not v.is_passed():
-                report_str += "# %s\n\n%s\n---\n" % (k, v)
+                generated_report = str(v)
+                if len(generated_report):
+                    if self.args.todo_only:
+                        report_str += "* %s:\n%s" % (k, generated_report)
+                    else:
+                        report_str += "# %s\n\n%s\n---\n" % (k, generated_report)
```

Comment:
hm, this becomes a bit convoluted. Let's see. How about:

```suggestion
                if not len(generated_report):
                    continue
                report_str += "* %s:\n%s" % (k, generated_report) if self.args.todo_only else "# %s\n\n%s\n---\n" % (k, generated_report)
```

Reply:
done